contrib/python-pybind11/patches/010-pybind11-remove-unwanted-pip-build-deps.patch

```diff
@@ -0,0 +1,9 @@
+--- a/pyproject.toml
++++ b/pyproject.toml
+@@ -1,5 +1,5 @@
+ [build-system]
+-requires = ["setuptools>=42", "cmake>=3.18", "ninja"]
++requires = ["setuptools>=42"]
```

Comment:
does not removing this hurt anything since you install them anyway to build the tests?

Reply:
it errored out without the patch, will try again